### PR TITLE
 Add rectangle

### PR DESCRIPTION
**Pull request by ManarOmari**
add rectangle and test setup html and css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
                
                <div class="header-navbar-image">
                    <div class="flex-container-img">
-                    <img src="assets\images\toolbar-mobile.png">
+                    <img src="assets/images/toolbar-mobile.png" alt="image icon logo">
```
Comment (danmainah) [inline index.html:19]: - [x] Kindly remove the image logo from your nav tag, only the hamburger section should have a nav tag, also make sure your logo has a link `<a>` tag.

Comment (danmainah) [reply index.html:19]: - [x] Kindly make sure at this stage you only have a hamburger menu in your nav items, as the project is beginning with a mobile version, the other sub-menus will be implemented later.

File: index.html [hunk 2]
```diff
@@ -54,13 +58,13 @@ <h1>I’m Alexandra Glad to see you!</h1>
        </div>
```
Comment (danmainah) [inline index.html:58]: - [x] Kindly make sure your button, has an an`<a>` tag or a `<button>` tag and it follows the design given.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ManarOmari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️

- [ ] Kindy make sure your design matches the design given, below is a comparison between your design and the given design;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

- [x] Kindly add the getting started section in your readme file, I can see the subtitles already added.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ManarOmari** [comment]
I closed this pull request by mistake and i cant to reopen it again

**Pull request merged**